webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const TerserPlugin = require('terser-webpack-plugin');
+
+const path = require('path');
+const package = require('./package.json');
+
+module.exports = {
+  devtool: "nosources-source-map",
+  entry: './src/lob-address-elements.js',
+  optimization: {
+    minimizer: [
```

Comment:
I'm assuming es6 is the default output? We might consider IE11+ as the min target to better handle the range of legacy browser clients we might see from customers on a site like Shopify.

Reply:
The default output is ES5 which is supported by IE10+. See the [ecma minify option here](https://terser.org/docs/api-reference.html#compress-options).